_data/chains/eip155-61803.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Etica Mainnet",
+  "chain": "Etica Protocol (ETI/EGAZ)",
+  "icon": "etica",
+  "rpc": ["https://eticamainnet.eticascan.org", "https://eticamainnet.eticaprotocol.org"],
+  "faucets": ["http://faucet.etica-stats.org/"],
+  "nativeCurrency": {
+    "name": "EGAZ",
+    "symbol": "EGAZ",
+    "decimals": 18
+  },
+  "infoURL": "https://eticaprotocol.org",
+  "shortName": "Etica",
+  "chainId": 61803,
+  "networkId": 61803,
+
+  "explorers": [
+    {
+      "name": "eticascan",
+      "url": "https://eticascan.org",
+      "standard": "EIP3091"
+    },
+    {
+      "name": "eticastats",
+      "url": "http://explorer.etica-stats.org/token/0x34c61EA91bAcdA647269d4e310A86b875c09946f",
```

Comment:
```suggestion
      "url": "http://explorer.etica-stats.org",
```

Reply:
Thanks! SHould I redo it or it's ok? Also I couldn't find how to convert a png into a binary image in iconDownload, if you tell me how I will do it with pleasure